src/pages/index.js

```diff
@@ -49,30 +123,70 @@ const enableValidation = (settings) => {
 
 enableValidation(settings);
 
-// Section class Instance
+/*-----------------------------------------------------------------*/
+/*             Class Instances                          */
+/*-----------------------------------------------------------------*/
+
+function handleSubmit(request, popupInstance) {
+  popupInstance.renderLoading(true);
+  //popupInstance.setSubmitButtonText("Saving...")
+  request()
+    .then(() => {
+      popupInstance.close();
+    })
+    .catch((error) => console.error(error))
+    .finally(() => {
+      popupInstance.renderLoading(false);
+      //popupInstance.setSubmitButtonText("Save")
+    });
+}
+
+// Section instance
 const cardSection = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: (data) => {
-      cardSection.addItem(createCard(data));
+      cardSection.addItem(createCard(data), "append");
     },
   },
   cardContainer
 );
-cardSection.renderItems();
 
-// UserInfo class instance
-const userInfo = new UserInfo({
-  profileName: "#profile_name",
-  profileJob: "#profile_job",
+// Edit Avatar instance
+const popupWithAvatarForm = new PopupWithForm(
+  "#edit-avatar-modal",
+  (formData) => {
+    handleSubmit(() => {
+      return api.changeProfilePic(formData.avatar).then((data) => {
+        userInfo.setUserInfo(data);
+        popupWithAvatarForm.close();
```

Comment:
`        popupWithAvatarForm.close();
`

`handleSubmit` has the closing, so you don't need it here